lib/syskit/network_generation/logger.rb

```diff
@@ -82,6 +82,21 @@ def configure
                 end
             end
 
+            @@logfile_indexes = {}
+
+            def self.default_log_file_name(name)
+                name[/.*(?=_[L|l]ogger)/] || name
+            end
+
+            def self.setup_default_logger(logger, log_file_name: default_log_file_name(logger.basename))
```

Comment:
`basename` is an artefact of orocos.rb. Within Syskit, just use `logger.orocos_name`